python_ta/__init__.py

```diff
@@ -19,10 +19,15 @@
 # First, remove underscore from builtins if it has been bound in the REPL.
 import builtins
 
-from pylint.config.config_file_parser import _ConfigurationFileParser
-from pylint.config.exceptions import _UnrecognizedOptionError
 from pylint.lint import PyLinter
 
+from python_ta.config import (
```

Comment:
You should be able to do a relative import here, something like `from .config import ...` (https://www.geeksforgeeks.org/absolute-and-relative-imports-in-python/)